### PR TITLE
918204 - Changes added for the resource texts used for localization

### PR DESCRIPTION
**Pull request by Kerubananthan3761**
## Description

Replaced Google translated strings with ChatGPT provided values for the localization texts in .resx file.

### CODE REVIEW

**Review by Jeyasri-Murugan — APPROVED** [review]

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx [hunk 1]
```diff
@@ -122,111 +122,111 @@
...
+    <value>Décembre</value>
     <comment>Text for December</comment>
   </data>
   <data name="Delete" xml:space="preserve">
     <value>Supprimer</value>
     <comment>Content for Delete button</comment>
   </data>
   <data name="DeleteOccurrence" xml:space="preserve">
-    <value>Supprimer cette occurrence</value>
+    <value>Supprimer l'occurrence</value>
```
Comment (Karthickmani97) [inline Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx:177]: Update this `Supprimer l'occurrence`

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx [hunk 2]
```diff
@@ -242,35 +242,35 @@
     <comment>Text for Hour</comment>
   </data>
   <data name="HoursIntervalTypeReminder" xml:space="preserve">
-    <value>Heures</value>
+    <value>Intervalle d'heures pour le rappel</value>
```
Comment (Karthickmani97) [inline Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: @Kerubananthan3761 , Can re-check and confirm this?

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx [hunk 3]
```diff
@@ -242,35 +242,35 @@
...
     <comment>Text for Hours</comment>
   </data>
   <data name="InValidEndTimeMessageAlert" xml:space="preserve">
-    <value>La date de fin que vous avez entrée se produit avant la date de début.</value>
+    <value>Alerte de message de fin non valide</value>
```
Comment (Karthickmani97) [inline Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: Check this also

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx [hunk 4]
```diff
@@ -242,35 +242,35 @@
...
     <comment>Message alert for invalid end time</comment>
   </data>
   <data name="InvalidOccurrenceEditingMessageAlert" xml:space="preserve">
-    <value>Deux occurrences de [] ne peuvent pas se produire le même jour.</value>
+    <value>Alerte de message d'édition d'occurrence non valide</value>
```
Comment (Karthickmani97) [inline Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: Check this

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx [hunk 5]
```diff
@@ -242,35 +242,35 @@
...
     <comment>Text for July</comment>
   </data>
   <data name="June" xml:space="preserve">
-    <value>juin</value>
+    <value>Juin</value>
     <comment>Text for June</comment>
   </data>
   <data name="JustThisOne" xml:space="preserve">
-    <value>Juste celui-là</value>
+    <value>Seulement celle-ci</value>
```
Comment (Karthickmani97) [inline Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx:269]: Check this and confirm

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx [hunk 6]
```diff
@@ -290,19 +290,19 @@
     <comment>Text for Minutes</comment>
   </data>
   <data name="MinutesIntervalTypeReminder" xml:space="preserve">
-    <value>Minutes</value>
+    <value>Intervalle de minutes pour le rappel</value>
```
Comment (Karthickmani97) [inline Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: Check

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx [hunk 7]
```diff
@@ -314,31 +314,31 @@
...
     <comment>Text for October</comment>
   </data>
   <data name="Ok" xml:space="preserve">
-    <value>D'accord</value>
+    <value>Ok</value>
     <comment>Content for Ok button that is displayed when edit/delete recurring appointment</comment>
   </data>
   <data name="OutRangeOccurrencedEditingMessageAlert" xml:space="preserve">
-    <value>Impossible de replanifier une occurrence du rendez-vous récurrent [] s'il ignore une occurrence ultérieure du même rendez-vous.</value>
+    <value>Alerte de message d'édition d'occurrence hors plage</value>
```
Comment (Karthickmani97) [inline Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: Check

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx [hunk 8]
```diff
@@ -354,47 +354,47 @@
...
     <comment>Text for Saturday</comment>
   </data>
   <data name="Save" xml:space="preserve">
-    <value>sauver</value>
+    <value>Enregistrer</value>
     <comment>Text for Save</comment>
   </data>
   <data name="Second" xml:space="preserve">
-    <value>Seconde</value>
+    <value>Deuxième</value>
```
Comment (Karthickmani97) [inline Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: seconde - this value showing in chatGPT, can check and confirm ?

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx [hunk 9]
```diff
@@ -354,47 +354,47 @@
...
     <comment>Text for Second</comment>
   </data>
   <data name="September" xml:space="preserve">
-    <value>septembre</value>
+    <value>Septembre</value>
     <comment>Text for September</comment>
   </data>
   <data name="Snooze" xml:space="preserve">
-    <value>Roupillon</value>
+    <value>Répéter</value>
```
Comment (Karthickmani97) [inline Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx:385]: Check

**Review by Karthickmani97 — COMMENTED** [review]
@Kerubananthan3761 , Address review concerns

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx (hunk 1 above)
Comment (Kerubananthan3761) [reply Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx:177]: Checked, getting same text Supprimer l'occurrence
image

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx (hunk 2 above)
Comment (Kerubananthan3761) [reply Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: Checked, getting same text as **Intervalle d'heures pour le rappel**

image

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx (hunk 3 above)
Comment (Kerubananthan3761) [reply Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: Changed text to **Alerte de message : heure de fin non valide**
image

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx (hunk 4 above)
Comment (Kerubananthan3761) [reply Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: Changed text to **Alerte : l’édition de l’occurrence est invalide**

image

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx (hunk 5 above)
Comment (Kerubananthan3761) [reply Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx:269]: Checked, getting same text
image

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx (hunk 6 above)
Comment (Kerubananthan3761) [reply Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: Checked, getting same text
image

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx (hunk 7 above)
Comment (Kerubananthan3761) [reply Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: Changed text to **Alerte de message : occurrence hors de la plage**

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx (hunk 8 above)
Comment (Kerubananthan3761) [reply Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: I have used all scheduler related terms and checked at once got **Deuxième**. Changed text to **Seconde**

image

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx (hunk 9 above)
Comment (Kerubananthan3761) [reply Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx:385]: Checked, getting same text
image

File: Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx [hunk 10]
```diff
@@ -246,11 +246,11 @@
     <comment>Text for Hours</comment>
   </data>
   <data name="InValidEndTimeMessageAlert" xml:space="preserve">
-    <value>Alerte de message de fin non valide</value>
+    <value>Alerte de message : heure de fin non valide</value>
```
Comment (Karthickmani97) [inline Resources/Syncfusion.SfScheduler.WPF.fr-FR.resx]: @Kerubananthan3761 , i am getting below text

image

**Review by Karthickmani97 — APPROVED** [review]

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**